static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -326,6 +349,37 @@ class EventsTable extends Component<Props, State> {
         }
         return col;
       });
+
+    const joinCustomData = ({data}: TableData) => {
+      if (this.state.attachments.length) {
+        const {projectId} = this.props;
+        const attachmentsWithUrl = this.state.attachments.map(attachment => ({
+          ...attachment,
+          url: `/api/0/projects/${organization.slug}/${projectId}/events/${attachment.event_id}/attachments/${attachment.id}/`,
+        }));
+        const eventIdMap = {};
+        data.forEach(event => {
+          event.attachments = [] as any;
+          eventIdMap[event.id] = event;
+        });
+        attachmentsWithUrl.forEach(attachment => {
+          const eventAttachments = eventIdMap[attachment.event_id]?.attachments;
+          if (eventAttachments) {
+            eventAttachments.push(attachment);
+          }
+        });
+      }
+    };
+
+    const fetchAttachments = async ({data}: TableData, cursor: string) => {
+      const eventIds = data.map(value => value.id);
+      const eventIdQuery = `event_id=${eventIds.join('&event_id=')}`;
+      const res = await this.api.requestPromise(
+        `/api/0/issues/${this.props.issueId}/attachments/?${eventIdQuery}`
+      );
```

Comment:
This creates _very_ long URLs, like 1,000 characters for 25 events! I'm assuming the API folks okayed this since the call is pretty fast (200ms or so?) so, this is just a heads-up

Reply:
Yeah, I also had that concern. but I was okayed it as long as it doesn't get any `414 request too long` in prod